### ISSUE TITLE
ERROR : No video formats found 

### ISSUE THREAD

**Issue by p0lux** [rojter-tech/pluradl.py#59]
### Hello !

Sometimes I get a bug with the video format. It doesn't happen on all formations. Is there a solution for this problem?

ERROR No video formats found; please report this issue on https://github.com/rojter-tech/pluradl.py/issues . Make sure you are using the latest version; see  https://github.com/rojter-tech/pluradl.py/wiki  on how to update. Be sure to call plura-dl with the --verbose flag and include its complete output.

**Comment by rojter-tech** [comment]
@p0lux Did you follow the recommendation to double check if your subscription covers the specific courses that causes this exception?

**Comment by p0lux** [comment]
Yes, my subscription covers specific courses. This bug occurs in the middle of the download. However I tried to restart the script on the same formation and the bug didn't appear. 
This bug appeared to me a few times in the last few days, but at each relaunch it was no longer present on the impacted videos.

**Comment by Taranis01** [comment]
Maybe your connection is not stable? I got the same errors when using a VPN that either reconnected (new IP) or was not able to connect (with Kill-Switch)
